<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>FileBox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/images/images.qrc">
    <normaloff>:/images/FileBox_1.png</normaloff>:/images/FileBox_1.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-size:18px;
	font-family: Trajan;
}

QFrame{
	background:#333;
	border-radius:15px;
}

#Login{
	background:url(:/images/bg.png);
}

QLabel{
	color:white;
	background:transparent;
}

QPushButton{
	color:white;
	background:red;
	border-radius:15px;
}

QPushButton:hover{
	background:#800000;
	color:#333
}

QLineEdit{
	background:transparent;
	border:none;
	color:#717072;
	border-bottom:1px solid #717072;
}

#lblLogin{
	color: white;
	font-size: 26px;
}

#lblWarning{
	color: red;
}

#lblWait{
	color: grey;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>480</width>
     <height>450</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>450</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblLogin">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>30</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRegister">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>REGISTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUsername">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leIP">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWarning">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblWait">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
